src/containers/AnswerQuestionContainer.tsx

```diff
@@ -113,14 +117,46 @@ const AnswerQuestionContainer = (props: IProps & RouteComponentProps) => {
       });
   };
 
+  const { questionText, title, answerText } = question;
+
+  const onPublishQuestion = event => {
+    if (!id) return;
+    publishQuestion(id)
+      .then(() => {
+        setModalText('Svaret er nå publisert!');
+        setHideModalButtons(true);
+      })
+      .catch(() => {
+        setHideModalButtons(true);
+        setModalText('Noe gikk galt.');
+      });
+    setTimeout(() => history.goBack(), 3000);
+    event.preventDefault();
+  };
+
+  const onDontPublish = event => {
+    setHideModalButtons(true);
+    setModalText(
+      'Svaret er sendt til eleven, men ble ikke publisert på Digitalleksehjelp.no',
+    );
+    setTimeout(() => history.goBack(), 3000);
+    event.preventDefault();
+  };
+
   return (
     <div>
+      {/*{
+        modalVisible && <Modal text={modalText}  isPublish={isPublish} isModalOpen={setModalVisible} id={id}/>
+      }*/}
       {modalVisible && (
         <Modal
-          text={modalText}
-          isPublish={isPublish}
-          isModalOpen={setModalVisible}
-          id={id}
+          content={modalText}
+          successButtonText={isPublish ? 'Publiser svaret' : 'Avbryt'}
+          warningButtonText={isPublish ? 'Ikke publiser' : 'Slett'}
```

Comment:
Har forsåvidt ikke noe slett funksjonalitet enda, så det kan fjernes herfra, når det ikke er i modalen :)

Reply:
Oki, da gjør jeg 
```suggestion
          warningButtonText={isPublish && 'Ikke publiser' }
```

istedenfor?

Reply:
Ja, eller bare:
```suggestion
          successButtonText={'Publiser svaret'}
          warningButtonText={'Ikke publiser'}
```